src/main/java/EigenValue.java

```diff
@@ -0,0 +1,59 @@
+public class EigenValue {
+
+    public static void main(String[] args) {
+        int[][] matrix = new int[][]{{2, -1}, {-1, 2}};
+        getEigenValues(matrix);
+    }
+
+    public static void getEigenValues(int[][] matrix) {
+        validateMatrix(matrix);
+        int b = calculateBForEquation(matrix);
+        double determinant = solveDeterminant(b, calculateCForEquation(matrix));
+        double[] result = solveQuadraticEquation(determinant, b);
+        printEigenValues(result);
+    }
+
+    public static void validateMatrix(int[][] matrix) {
```

Comment:
There are many problems with this function:
 * You don't know how it gives you the result of the validation
 * If with exception, you should use something more specific than Runtime. It conveys no message. Probably it's IllegalArgumentException what you are looking for.
* The application shouldn't die because you passed in something non-sense. Think about it, this way the return code will be 1, whereas you just wrote code to handle it. So it is expected :)